src/talker_character.h

```diff
@@ -32,6 +32,14 @@ class talker_character_const: public talker_cloner<talker_character_const>
         }
         ~talker_character_const() override = default;
 
+        const Character *get_character() const override {
+            return me_chr_const;
+        }
+
+        Character *get_character() override {
+            return nullptr;
+        }
+
```

Comment:
```suggestion
```
This part is not needed anymore (added in [#68401](https://github.com/CleverRaven/Cataclysm-DDA/pull/68401/files))